terraform/dns.tf

```diff
@@ -3,8 +3,11 @@
 #
 resource "aws_route53_zone" "internal" {
   name       = "${var.r53_private_hosted_zone}"
```

Comment:
After the changes below, `terraform fmt` adjusts the number of spaces before the `=`.